pkg/primitive/return.go

```diff
@@ -0,0 +1,93 @@
+package primitive
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hyperifyio/gnd/pkg/log"
+)
+
+// Return represents the return primitive
+type Return struct {
+	// IsSubroutine indicates whether this primitive is being executed in a subroutine context
+	IsSubroutine bool
+	// Destination is the destination for the return operation
+	Destination string
```

Comment:
We don't need to control destination either. What we need is the optional return value only, and maybe an indication that this is a return result.